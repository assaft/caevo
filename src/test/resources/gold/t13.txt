John left between Monday and Wednesday.