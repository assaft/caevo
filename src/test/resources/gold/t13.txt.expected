ei1,left
t1,1969-07-21
t2,1969-07-23
l1,ei1,t1,OVERLAP_OR_AFTER,AdjacentVerbTimex
l2,ei1,t2,BEFORE_OR_OVERLAP,AdjacentVerbTimex
l3,t1,t2,BEFORE,TimeTimeSieve